docs/content/manual/backup-and-restore/4907-vm-backup-restore-longhorn-1.5-plus.md

```diff
@@ -0,0 +1,68 @@
+---
+title: VM backup restore use Longhorn v1.5+ attach/detach mechanism
+---
+
+* Related issues: [#4907](https://github.com/harvester/harvester/issues/4907) [ENHANCEMENT] Take advantage of the attach/detach mechanism in Longhorn v1.5+
+
+## Category: 
+* Backup and Restore
+
+## Verification Steps
+Case 1: snapshot can work on a stopped VM.
+1. Create a VM.
+1. After the VM is ready, stop the VM.
+1. Check VM volumes are detached.
+1. Take a snapshot on the VM. The snapshot can be ready.
+
+Case 2: restore a snapshot from detached volumes can work.
+1. Follow Case 1.
+1. Make sure VM volumes are detached.
+1. Restore the snapshot to a new VM. The new VM can be ready.
+1. Restore the snapshot to replace the old VM. The old VM can be ready.
+
+Case 3: backup can work on a stopped VM
```

Comment:
Same as above, can you check if we have this in automated test?

Reply:
For Case 2: 
I found the following test plan in `test_4_vm_snapshot.py` integration test may covered in the test `test_restore_from_vm_snapshot_while_pvc_detached_from_source` 

https://github.com/harvester/tests/blob/790c494fceba390253d8130641aa5db5d0289096/harvester_e2e_tests/integrations/test_4_vm_snapshot.py#L425-L453

Reply:
Update the test plan, remove the lines that have been covered in the automation e2e test. 
And replace to indicate which e2e test covered this line.